locksmith/src/utils/middlewares/isVerifierMiddleware.ts

```diff
@@ -52,3 +59,37 @@ export const isVerifierMiddleware: RequestHandler = async (req, res, next) => {
     })
   }
 }
+
+export const isEventVerifierOrManagerMiddleware: RequestHandler = async (
+  req,
+  res,
+  next
+) => {
+  try {
+    const eventSlug = req.params.eventSlug
+    const address = Normalizer.ethereumAddress(req.user!.walletAddress!)
+
+    const eventVerifiers = await getEventVerifiers(eventSlug)
+
+    const isVerifier =
+      eventVerifiers
+        ?.map((item) => Normalizer.ethereumAddress(item.address))
+        .includes(Normalizer.ethereumAddress(address)) ||
+      (await isEventOrganizer(address, eventSlug))
```

Comment:
Since this is an ENUM this won't return a boolean here and the logic breaks as this will always be truthy.